controlplane/test/persisted-operations.test.ts

```diff
@@ -36,141 +33,143 @@ const setupFederatedGraph = async (fedGraphName: string, client: Client) => {
-      });
+    await server.close();
+  });
 
-      expect(publishOperationsResp.response?.code).not.toBe(EnumStatusCode.OK);
+  test('Should not publish persisted operations with an invalid federated graph name', async (testContext) => {
+    const { client, server } = await SetupTest(testContext, dbname);
+    const fedGraphName = genID('fedGraph');
+    await setupFederatedGraph(fedGraphName, client);
 
-      await server.close();
+    const publishOperationsResp = await client.publishPersistedOperations({
+      fedGraphName: `not_${fedGraphName}`,
+      namespace: 'default',
+      operations: [{ id: 'hello4', contents: `query { does_not_exist }` }],
     });
 
-    test('Should store persisted operations in blob storage', async (testContext) => {
-      const { client, server, blobStorage } = await SetupTest(testContext, dbname);
-      const fedGraphName = genID('fedGraph');
-      await setupFederatedGraph(fedGraphName, client);
-
-      const id = genID('hello');
-      const query = `query { hello }`;
-
-      const publishOperationsResp = await client.publishPersistedOperations({
-        fedGraphName,
-        clientName: 'test-client',
-        operations: [{ id, contents: query }],
-      });
-
-      expect(publishOperationsResp.response?.code).toBe(EnumStatusCode.OK);
-
-      const storageKeys = blobStorage.keys();
-      expect(storageKeys.length).toBe(2);
-      const keyComponents = storageKeys[1].split('/');
-      const keyFilename = keyComponents.at(-1)!;
-      const keyBasename = keyFilename.split('.')[0];
-      expect(keyBasename).toBe(id);
-
-      const stream = await blobStorage.getObject(storageKeys[1]);
-      const text = await new Response(stream).text();
-      expect(JSON.parse(text)).toEqual({ version: 1, body: query });
-      await server.close();
+    expect(publishOperationsResp.response?.code).not.toBe(EnumStatusCode.OK);
+
+    await server.close();
+  });
+
+  test('Should store persisted operations in blob storage', async (testContext) => {
+    const { client, server, blobStorage } = await SetupTest(testContext, dbname);
+    const fedGraphName = genID('fedGraph');
+    await setupFederatedGraph(fedGraphName, client);
+
+    const id = genID('hello');
+    const query = `query { hello }`;
+
+    const publishOperationsResp = await client.publishPersistedOperations({
+      fedGraphName,
+      namespace: 'default',
+      clientName: 'test-client',
+      operations: [{ id, contents: query }],
     });
 
-    test('Should delete persisted operations from blob storage when the federated graph is deleted', async (testContext) => {
-      const { client, server, blobStorage } = await SetupTest(testContext, dbname);
-      const fedGraphName = genID('fedGraph');
-      await setupFederatedGraph(fedGraphName, client);
+    expect(publishOperationsResp.response?.code).toBe(EnumStatusCode.OK);
+
+    const storageKeys = blobStorage.keys();
+    expect(storageKeys.length).toBe(2);
```

Comment:
Why was it needed to update so much?

Reply:
I am not sure whats going on here. Maybe some formatting or something reorderd, not sure. Shouldnt be a problem